### PR TITLE
To insert element in existing array

### PR DESCRIPTION
**Pull request by Skm26**
Added program to insert element into an existing array

### CODE REVIEW

File: java/Insert_array.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	a[pos-1]=x;
+	System.out.print("After inserting:");
+	for(int i=0;i<n;i++)
+	{
+	System.out.print(a[i]+",");
+	}
+	System.out.println(a[n]);
+}
+}
+
```
Comment (utkarsh-shekhar) [inline java/Insert_array.java:34]: Please remove excess lines.

File: java/Insert_array.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	public static void main(String[] args)
+	{
+	int n,pos,x;
+	Scanner s=new Scanner(System.in);
+	System.out.println("Enter no of elements you want in array:");
+	n=s.nextInt();
+	int a[]=new int[n+1];
+	System.out.println("Enter alll the elements:");
+	for(int i=0;i<n;i++)
+		{
```
Comment (utkarsh-shekhar) [inline java/Insert_array.java:13]: Please format the code.

If the IDE you use doesn't support that, it's time to get one that does. ;)
You can also use online tools for formatting code, like: [https://codebeautify.org/javaviewer](https://codebeautify.org/javaviewer)

**Review by utkarsh-shekhar — CHANGES_REQUESTED** [review]

**Comment by utkarsh-shekhar** [comment]
Also, the file name should either be snake case or camel case or Pascal case.

**Review by Skm26 — COMMENTED** [review]
Changes Done

**Comment by Skm26** [comment]
Have done the Changes,Please do add the hacktober fest tag.
Thank you

**Review by utkarsh-shekhar — APPROVED** [review]
Nice work!

**Pull request merged**